easy-ui-react/src/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,160 @@
+@use "../styles/common" as *;
+
+.Checkbox {
+  @include component-token("checkbox", "size", 16px);
+  @include component-token("checkbox", "box-margin-top", 4px);
+  @include component-token("checkbox", "text-margin-ver", 0);
+  @include component-token(
+    "checkbox",
+    "border-radius",
+    design-token("shape.border_radius.sm")
+  );
+  @include component-token("checkbox", "gap", design-token("space.1"));
+  @include component-token(
+    "checkbox",
+    "color.mark",
+    theme-token("color.text.primary.inverse")
+  );
+  @include component-token("checkbox", "color.background", transparent);
+  @include component-token(
+    "checkbox",
+    "color.border",
+    theme-token("color.border.boldest")
+  );
+  @include component-token(
+    "checkbox",
+    "color.error-icon",
+    theme-token("color.text.danger")
+  );
+}
+
+.Checkbox,
+.label {
+  display: inline-flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  gap: component-token("checkbox", "gap");
+}
+
+.box,
+.errorIcon {
+  flex: 0 0 auto;
+}
+
+.box,
+.mark {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: component-token("checkbox", "size");
+  height: component-token("checkbox", "size");
+}
+
+.box {
+  background: component-token("checkbox", "color.background");
+  border: design-token("shape.border_width.1") solid
+    component-token("checkbox", "color.border");
+  border-radius: component-token("checkbox", "border-radius");
+  margin-top: component-token("checkbox", "box-margin-top");
+
+  .focusVisible & {
+    @include native-focus-ring;
+  }
+}
+
+.mark {
+  color: component-token("checkbox", "color.mark");
+}
+
+.text {
+  display: inline-flex;
+  margin: component-token("checkbox", "text-margin-ver") 0;
+}
+
+.errorIcon {
+  display: inline-flex;
+  color: component-token("checkbox", "color.error-icon");
+}
+
+.indeterminateSvg {
+  position: absolute;
+}
+
+.nested {
+  @include component-token("checkbox", "text-margin-ver", 4px);
+}
+
+.sizeLg {
+  @include component-token("checkbox", "size", 24px);
+  @include component-token("checkbox", "box-margin-top", 0);
+  @include component-token("checkbox", "gap", design-token("space.1.5"));
+}
```

Comment:
**question**: do you know whats the use case for the large size variation? also is it possible (or valid) to nest checkboxes with different sizes?

Reply:
i waffled some on this. it's not formally in the `Checkbox` specs but a larger sized checkbox is used here in the `Checked Card` variant so i created the `lg` size to accommodate it:

<img width="529" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/80e1dbdf-25ee-48b8-a7fd-fed8dc0a6446">

i purposefully didn't document it too much in Storybook since that size isn't officially spec'd.

let me know if you think i should revisit that.

Reply:
ah got it, makes sense